.github/stale.yml

```
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

We don't use either of these labels. Can we change this to be `bug`? I know bugs shouldn't last 60 days but I wouldn't want them to be closed just because no one has gotten to them. I'm fine to not do this, but its a suggestion.

```suggestion
  - bug
```